packages/http/src/util/__tests__/runtimeExpression.spec.ts

```diff
@@ -0,0 +1,120 @@
+import { resolveRuntimeExpression, resolveRuntimeExpressions } from '../runtimeExpression';
+import { assertNone, assertSome } from '@stoplight/prism-core/src/utils/__tests__/utils';
+
+describe('resolveRuntimeExpression', () => {
+  it('resolves $method', () => {
+    assertSome(
+      resolveRuntimeExpression('$method', { method: 'get', url: { path: '' } }, { statusCode: 200 }),
+       value => expect(value).toEqual('get'),
+    );
+  });
+
+  it.skip('resolves $url', () => {
```

Comment:
Aha! You thought I wouldn't see this? :trollface: 

You bad boy

![](https://media.giphy.com/media/cJAvY9MCwFwBjESRk0/giphy.gif)